src/Components/Patient/PatientHome.tsx

```diff
@@ -332,7 +332,10 @@ export const PatientHome = (props: any) => {
 
   if (isConsultationLoading) {
     consultationList = <CircularProgress size={20} />;
-  } else if (consultationListData.length === 0) {
+  } else if (
+    consultationListData.length === 0 ||
+    (consultationListData.at(0)?.discharge_date && patientData.is_active)
```

Comment:
@cp-Coder 
```suggestion
    (consultationListData.at(0)?.discharge_date && patientData.is_active)
```

shouldn't this `.at(0)` be the last consultation??

Reply:
@khavinshankar, Yes it is the last consultation. If the patient is discharged, the last consultation will have a discharge date.